src/01-general/users.apib

```diff
@@ -48,6 +48,7 @@ Get a list of all users.
     + Attributes (object)
         + filter (object, optional)
             + ids: `cb8da52a-ce89-4bf6-8f7e-8ee6cb85e3b5`,`f8a57a6f-dd1e-41a3-b8d3-428663f1d09e` (array[string], optional)
+            + term: `John` (string, optional) - Filters on username, real name and email address
```

Comment:
B: there is no notion of "real name" in the return data for users. The first and last name is returned as a result from splitting on the first space.

```suggestion
            + term: `John` (string, optional) - Filters on username, first_name, last_name and email
```

Also the username still needs to be added to the user info.